src/include/index/vamana_index.h

```diff
@@ -95,7 +96,8 @@ auto greedy_search(
     auto&& query,
     size_t k_nn,
     size_t L,
-    Distance distance = Distance{}) {
+    Distance distance = Distance{},
+    bool convert_to_db_ids = false) {
```

Comment:
Why do we need `convert_to_db_ids`? The queries should always return the external_ids or the default ids if external ids were not provided